packages/web/src/shared/components/form/SelectInput.tsx

```diff
@@ -8,17 +8,15 @@ export interface SelectInputProps
 }
 
 const SelectInput = forwardRef<HTMLSelectElement, SelectInputProps>(
-  (
-    { className = '', name, onChange, onBlur, required, children, ...props },
-    ref
-  ) => {
+  ({ className = '', required, children, ...props }, ref) => {
     const context = useFormContext();
-    const hasErrors = !!context?.formState.errors[name];
-    const registerProps = context?.register(name, {
+    const hasErrors = !!context?.formState.errors[props.name];
+    const registerProps = context?.register(props.name, {
       required,
-      onChange,
-      onBlur,
+      onChange: props.onChange,
+      onBlur: props.onBlur,
```

Comment:
This is what broken the SelectInput. I was destructuring props to get the change and blur handlers for the validation library, but when there is no validation these, still need to be attached to the input element